### ISSUE TITLE
Git amend produces 500 error

### ISSUE THREAD

**Issue by DevelPaga** [microsoft/vscode#154463]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.69.0
- OS Version: Darwin x64 21.5.0

Steps to Reproduce:

1. Create a commit
2. Add some changes
3. Commit amend the commit (... > Commit > Commit All (Amend) )

Error code:
```
> git -c user.useConfigOnly=true commit --quiet --amend
Error: Bad status code: 500
    at ClientRequest.<anonymous> (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/git-editor-main.js:1:470)
    at Object.onceWrapper (node:events:510:26)
    at ClientRequest.emit (node:events:390:28)
    at HTTPParser.parserOnIncomingClient (node:_http_client:623:27)
    at HTTPParser.parserOnHeadersComplete (node:_http_common:128:17)
    at Socket.socketOnData (node:_http_client:487:22)
    at Socket.emit (node:events:390:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at Socket.Readable.push (node:internal/streams/readable:228:10)
error: There was a problem with the editor '"/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/git-editor.sh"'.
Please supply the message using either -m or -F option.
```

Running `git commit --amend` from the command line works

**Comment by makz159** [comment]
> Does this issue occur when all extensions are disabled?: Yes
> 
> * VS Code Version: 1.69.0
> * OS Version: Darwin x64 21.5.0
> 
> Steps to Reproduce:
> 
> 1. Create a commit
> 2. Add some changes
> 3. Commit amend the commit (... > Commit > Commit All (Amend) )
> 
> Error code:
> 
> ```
> > git -c user.useConfigOnly=true commit --quiet --amend
> Error: Bad status code: 500
>     at ClientRequest.<anonymous> (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/git-editor-main.js:1:470)
>     at Object.onceWrapper (node:events:510:26)
>     at ClientRequest.emit (node:events:390:28)
>     at HTTPParser.parserOnIncomingClient (node:_http_client:623:27)
>     at HTTPParser.parserOnHeadersComplete (node:_http_common:128:17)
>     at Socket.socketOnData (node:_http_client:487:22)
>     at Socket.emit (node:events:390:28)
>     at addChunk (node:internal/streams/readable:315:12)
>     at readableAddChunk (node:internal/streams/readable:289:9)
>     at Socket.Readable.push (node:internal/streams/readable:228:10)
> error: There was a problem with the editor '"/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/git-editor.sh"'.
> Please supply the message using either -m or -F option.
> ```
> 
> Running `git commit --amend` from the command line works

see if #154449 helps you

**Comment by DevelPaga** [comment]
Yes, unchecking the `Git: Use Editor As Commit Input` setting fixes the issue

**Comment by lszomoru** [comment]
@DevelPaga, glad to hear that disabling `git.useEditorAsCommitInput` resolved this issue.

**Comment by lszomoru** [comment]
Closing this as duplicate of https://github.com/microsoft/vscode/issues/154449.
